main.go

```diff
@@ -43,88 +49,106 @@ func initClientConnection() bpb.BirthdayFunctionsClient {
 		log.Fatalln("connection error: ", err)
 		os.Exit(4)
 	}
-	// defer conn.Close()
```

Comment:
why did you delete this?

Reply:
i made mongo client connection in a different function which means that if im defering inside it, at the end it'll return the client and just close the connection any ways.